extensions/proposals/WEBGL_video_texture/extension.xml

```diff
@@ -20,21 +20,21 @@
   <overview>
     <mirrors href="https://www.khronos.org/registry/gles/extensions/OES/OES_EGL_image_external.txt"
              name="OES_EGL_image_external">
-      <addendum>Defines a new texture target <code>TEXTURE_EXTERNAL_OES</code>.</addendum>
-      <addendum>Provides a mechanism for binding <code>HTMLVideoElement</code>'s EGLImage to external texture targets.</addendum>
-      <addendum>Provides time of frame, texture width and height of <code>HTMLVideoElement</code>'s EGLImage.</addendum>
+      <addendum>Defines a new texture target <code>TEXTURE_VIDEO_IMAGE</code>.</addendum>
```

Comment:
For example, textureA is bound to TEXTURE_2D.

Now on a platform the underlying target for TEXTURE_VIDEO_IMAGE is TEXTURE_2D, so bindTexture(TEXTURE_VIDEO_IMAGE, textureB) will change TEXTURE_2D binding to textureB.

Now if a user tries to modify textureA by calling texImage2D(TEXTURE_2D, ...), then instead of modifying textureA, it's modifying textureB.